apps/chess-training-app/Dockerfile

```
@@ -0,0 +1,26 @@
+
+RUN yarn install --frozen-lockfile && \
+    yarn build
+
+# Production image
+FROM node:22-alpine
+WORKDIR /app
+
+ENV NODE_ENV=production
+
+COPY package.json yarn.lock ./
+RUN yarn install --production --frozen-lockfile
```

Hadolint (2.14.0)

[info] `yarn cache clean` missing after `yarn install` was run.

(DL3060)